### DISCUSSION TITLE
Iron Irwini Tutorial Testing party questions

### DISCUSSION THREAD

**Discussion by clalancette** [osrf/ros2_test_cases#1039] [General]
If you have questions about a particular tutorial, please leave comments in the individual issue; the admins will see them and respond.

If you have questions about the tutorial party itself, please leave a question here and the admins will field it here.

**Comment by SteveMacenski** [comment]
In the readme I see [cyclone, debian, amd64, jammy](https://github.com/osrf/ros2_test_cases/issues?q=is%3Aissue+is%3Aopen+label:%22cyclone%22+label:%22debian%22+label:%22amd64%22+label:%22jammy%22+label:%22generation-1%22) and [cyclone, binary, amd64, jammy](https://github.com/osrf/ros2_test_cases/issues?q=is%3Aissue+is%3Aopen+label:%22cyclone%22+label:%22binary%22+label:%22amd64%22+label:%22jammy%22+label:%22generation-1%22), which I think needs to be specified how to install it in both of those different ways + links to them. I colloquially refer to those interchangably, so its not immediately clear there's a semantic difference without further detail

**Reply by mjcarroll** [reply]
Binary refers to the binary distributions available here: https://github.com/ros2/ros2/releases/tag/release-iron-beta-20230502, where "debian" refers to the debian packages available via the apt mirror.

**Reply by SteveMacenski** [reply]
^ I ran into that, so I updated my remark :-)

**Reply by mjcarroll** [reply]
At this point, the naming is kind of historical, as that is how we have distinguished for past release.  I agree that it is a little ambiguous because debians (or rpms) are also binaries, but naming things is hard.

**Reply by cdhabecker** [reply]
Naming things is a little easier if you allow yourself to use more words. My suggestion: https://github.com/osrf/ros2_test_cases/issues/1046

**Comment by cdhabecker** [comment]
I can't assign myself to https://github.com/osrf/ros2_test_cases/issues/722. As per the instructions, I'll add a comment to the issue, instead. In the meantime, if there's a permission that you can grant to me, please do.

**Reply by mjcarroll** [reply]
As it turns out, this isn't a permission that we can grant in this case.  Commenting is the correct course of action, and one of the maintainers can assign you.

**Reply by cdhabecker** [reply]
Roger that, Michael. :-)

**Comment by SteveMacenski** [comment]
I could use some attention in https://github.com/osrf/ros2_test_cases/issues/1042#issuecomment-1533827714, I think I've uncovered something that needs further investigation.

**Reply by SteveMacenski** [reply]
I identified root cause and commit to revert to resolve it, but needs an OR person to make the call as to what the action is w.r.t. it https://github.com/osrf/ros2_test_cases/issues/1042

**Comment by cdhabecker** [comment]
Newbies are the most appropriate people to judge the effectiveness of the documentation; if a newbie can demonstrate the new information after reading the doc, then we know that the doc effectively transferred the information into the newbie's brain. Experts already have the information in their brains -- experts can pass the test even if the doc is ineffective.

**Comment by ChidiebereMicah** [comment]
Please, i would like to know what the difference between the binary install and Debian install is. This is because, in the installation tutorial, I see that the Debian install is under binary installation (the other option being fat) but they are both listed as different options for setup in the Jazzy test readme

**Reply by clalancette** [reply]
This is a great question, and one we haven't well clarified.  Can I ask you to please restate it on https://github.com/osrf/ros2_test_cases/discussions/1545 , which are questions for this year?  Then I'll answer over there.

**Reply by ChidiebereMicah** [reply]
Sure
